eblocker-icapserver/src/main/java/org/eblocker/server/common/data/Device.java

```diff
@@ -82,8 +82,13 @@ public class Device extends ModelObject {
     private boolean controlBarAutoMode = true;
     private boolean mobileState = true;
     private boolean mobilePrivateNetworkAccess;
+    private String lastSeen = "-";
+    private HashMap<String, Boolean> deviceLastSeenMap = new HashMap<String, Boolean>();
```

Comment:
I think this map is never used and should be removed. There are entries put into it but they are never read. Or am I missing anything?

Reply:
Forgot to delete it - thanks